api/v1alpha1/clusterextension_types.go

```diff
@@ -181,6 +186,8 @@ type ClusterExtensionStatus struct {
 	InstalledBundle *BundleMetadata `json:"installedBundle,omitempty"`
 	// +optional
 	ResolvedBundle *BundleMetadata `json:"resolvedBundle,omitempty"`
+	// +optional
+	Catalog string `json:"catalog,omitempty"`
```

Comment:
This makes me nervous. Is this the catalog for the installed bundle or the resolved bundle? They may have come from different catalogs.

My personal preference is to remove this field from the status for now and instead include something in the `message` of the `Resolved` and `Installed` conditions. And then in the future, if we see there is a need for a first-class field we'll understand that need better before we commit to something in a stable API.

Reply:
You are correct, I had updated the RFCs. I thought I removed this line.